README.md

```diff
@@ -62,14 +62,35 @@ print(mask.shape)
 > (256, 256)
 ```
 
-Create image from tile
+#### Create image from tile
 
 ```python
 from rio_tiler.utils import render
 
 buffer = render(tile, mask=mask) # this returns a buffer (PNG by default)
 ```
 
+Rescale non-byte data and apply colormap
+
+```python
+from rio_tiler.colormap import cmap
+from rio_tiler.utis import linear_rescale
+
+for bdx in range(tile.shape[0]):
```

Comment:
I think you should add a comment line here to make it clear what this is doing, e.g.
```
# Rescale each band linearly to the unsigned byte range of 0-255
```
Also do you have to provide `in_range` when you're rescaling? If not passed does it automatically find the `min/max` of the band?